webconsole/src/main/java/org/secondbase/webconsole/HttpWebConsole.java

```diff
@@ -78,15 +78,11 @@ public void start() throws IOException {
 
     @Override
     public void shutdown() throws IOException {
-        LOG.info("Shutting down webconsole.");
         if (! WebConsoleConfiguration.enableWebConsole) {
             return;
         }
-        final int USE_SYSTEM_DEFAULT_BACKLOG = 0;
-        server.bind(
-                new InetSocketAddress(port),
-                USE_SYSTEM_DEFAULT_BACKLOG);
-        server.start();
+        LOG.info("Shutting down webconsole.");
```

Comment:
Then logging has already shut down and I'll have to use System.out instead of logging.